### DISCUSSION TITLE
Vragen stellen in Dodona

### DISCUSSION THREAD

**Discussion by pdawyndt** [dodona-edu/feedback-and-support#6] [Feature requests]
Momenteel kunnen studenten via Dodona enkel vragen stellen bij een ingediende oplossing, en komt het antwoord als een afzonderlijk item binnen (niet gekoppeld aan een vraag). Lijkt me nuttig om ook te overwegen om studenten vragen te laten stellen op andere niveau's binnen een cursus (die je dan nog steeds kunt in/uitschakelen op cursusniveau):

- vraag over de cursus
- vraag over een reeks (is dit niveau nodig?)
- vraag over een oefening (bijvoorbeeld over de opgave, niet over een specifieke oplossing)

Er is ook betere opvolging mogelijk als vraag-en-antwoord in een discussiedraadje gebundeld blijven.

Voor studenten en lesgevers zou er dan ook een dashboard kunnen aangeboden worden waarin alle discussiedraadjes gebundeld worden, met filtering op scope (oefening, reeks, cursus), gebruiker (om communicatie met gebruiker in beeld te brengen) en inhoud (om vragen/antwoorden te kunnen terugvinden). 

Dat laatste kan ook een eerste aanzet zijn om tijdens het verbeteren makkelijk eerdere feedback terug te vinden. Wat ook handig kan zijn is het gebruik van hashtags in vragen/antwoorden als labels om zaken terug te vinden (bijvoorbeeld naam van een functie als label voor feedback op een bepaalde functie).

**Comment by chvp** [comment]
Issue voor threading: #2827. Het vragen-dashboard zou zeker aangepast moeten worden als we dit implementeren. Het openstellen naar studenten is nog een heel andere vraag, maar dat is zeker niet vanzelfsprekend. De meeste vragen worden op code gesteld, die over het algemeen niet door andere studenten gezien mag worden. Als de vraag/thread niet duidelijk is zonder die context zijn studenten hier niet veel mee.

Issue voor vragen op oefeningniveau: #2329. Persoonlijk zie ik het voordeel niet van een vraag op cursus-niveau, dit lijkt me eerder iets om via mail of een ander leerplatform te doen.

Ook voor het hergebruik van opmerkingen is er een issue: #2825. Hoe we dat hergebruik gaan vormgeven is er nog geen plan voor, voor zover ik weet. Een lijst van tags op een antwoord is dan zeker een optie.

**Reply by pdawyndt** [reply]
Met dashboard voor studenten doelde ik vooral op het aanbieden van een overzicht voor studenten, waarin ze makkelijk al hun eigen vragen (met bijhorende antwoorden) kunnen terugvinden. Dat is er momenteel niet.

**Reply by pdawyndt** [reply]
Ook gerelateerd aan deze discussie is #2355 met notificatie (voorstel is daar per email) voor lesgevers bij binnengekomen vragen die nog geen antwoord gekregen hebben. Een dashboard met alle vragen (cursusoverschrijdend) zou ook handig zijn voor lesgevers met meerdere cursussen. Op dit moment zie je immers enkel een notificatie van onbehandelde vragen op de cursuspagina. Dat zie je niet op je homepagina.

**Reply by chvp** [reply]
https://dodona.ugent.be/questions (gelinkt in gebruikersmenu) geeft studenten al een overzicht van hun eigen vragen. Ze kunnen daar ook op cursus filteren.

**Reply by chvp** [reply]
Overzicht voor lesgevers over alle cursussen heen is ook beschikbaar via drawer (maar inderdaad niet zichtbaar op homepagina).
